app/controllers/drawings_controller.rb

```diff
@@ -10,7 +13,7 @@ def index
     respond_to do |format|
       format.html
       format.js
-      format.json { render json: @drawings }
+      format.hal { respond_with drawings, represent_with: DrawingCollectionRepresenter }
```

Comment:
I'm not a big fan of this using the paginated version, but I think I just need to be educated on why we're using pagination, and how to use this successfully in the future.